docs/source/reference/mimetype.md

```diff
@@ -22,9 +22,11 @@ As types may contain vendor specific items, a
 - `application/vnd.jupyter.cells`
 - `application/vnd.jupyter.dragindex` used by nbdime
 - `application/x-ipynb+json` for notebooks
-- `text/html` has a metadata option, `'--isolated'` which can annotate an IPython display magic, `%%html`,
-  or be used as follows: `display(html, metadata={'text/html':{'isolated':True}})` Isolated cells are
-  rendered inside their own `<iframe>` tag.
+- `text/html`
+     - `'--isolated'`, metadata option, can be used as follows: 
+       `display(html, metadata={'text/html':{'isolated':True}})` and
+       can annotate an IPython display magic, `%%html`. Isolated cells are
+       rendered inside their own `<iframe>` tag.
```

Comment:
What I meant in #308 was that this should read:

```
- `text/html`
  * metadata:
    - `isolated: boolean` -- HTML should be rendered inside an `<iframe>`.
```

The IPython part should be documented in the IPython docs, not inside the mimetype docs for general jupyter use.

Reply:
Ah... that makes more sense.